### PR TITLE
Make codes more pythonic.

### PR DESCRIPTION
**Pull request by Kongsea**
Make some code parts more pythonic.

### CODE REVIEW

**Comment by robieta** [comment]
It looks like you missed an indent. (I'm in particular looking at train_image_classifier.py) If you did intend that clause to indented to be in the same if as `if var.op.name.startswith(exclusion)`, then the logic is different rather than just an aestetic change.

**Comment by Kongsea** [comment]
Thanks.
Now I have restored the indents.
Besides, I think the code logic is the same as before.

File: research/slim/train_image_classifier.py [hunk 1]
```diff
@@ -340,12 +340,10 @@ def _get_init_fn():
   # TODO(sguada) variables.filter_variables()
   variables_to_restore = []
   for var in slim.get_model_variables():
-    excluded = False
     for exclusion in exclusions:
       if var.op.name.startswith(exclusion):
-        excluded = True
         break
-    if not excluded:
+    else:
```
Comment (robieta) [inline research/slim/train_image_classifier.py]: This is the unmatched `else` I was referring to.

Comment (Kongsea) [reply research/slim/train_image_classifier.py]: No, it matches with **for**. It's a for...else... structure.

Comment (robieta) [reply research/slim/train_image_classifier.py]: Huh. TIL. That does indeed appear to be equivalent.

Let me get back to you tomorrow.

Comment (Kongsea) [reply research/slim/train_image_classifier.py]: Look forward to you.

**Pull request merged**